### PR TITLE
Update to the latest `config`

### PR DESCRIPTION
**Pull request by yevhenii-nadtochii**
This PR updates the repository to use the latest `config`.

The most significant changes are the following:

1.  [Bumped versions](https://github.com/SpineEventEngine/config/pull/344) of some dependencies.
2. `testJar` was [excluded](https://github.com/SpineEventEngine/config/pull/340) from the list of artifacts that are published by default.

The library version is bumped to `2.0.0-SNAPSHOT.88`.

### CODE REVIEW

**Comment by yevhenii-nadtochii** [comment]
@armiol Please take a look.

File: pom.xml [hunk 1]
```diff
@@ -32,25 +32,25 @@ all modules and does not describe the project structure per-subproject.
...
     <scope>compile</scope>
   </dependency>
   <dependency>
     <groupId>com.google.protobuf</groupId>
     <artifactId>protobuf-java</artifactId>
-    <version>3.19.3</version>
+    <version>3.19.4</version>
     <scope>compile</scope>
   </dependency>
   <dependency>
```
[the remark this answers is not in the document]
Comment (yevhenii-nadtochii) [reply pom.xml:44]: @sonatype-lift ignore

File: pom.xml [hunk 2]
```diff
@@ -32,25 +32,25 @@ all modules and does not describe the project structure per-subproject.
...
```
[the remark this answers is not in the document]
Comment (yevhenii-nadtochii) [reply pom.xml:32]: @sonatype-lift ignore

File: pom.xml [hunk 3]
```diff
@@ -32,25 +32,25 @@ all modules and does not describe the project structure per-subproject.
...
     <scope>compile</scope>
   </dependency>
   <dependency>
     <groupId>com.google.protobuf</groupId>
     <artifactId>protobuf-kotlin</artifactId>
-    <version>3.19.3</version>
+    <version>3.19.4</version>
     <scope>compile</scope>
   </dependency>
   <dependency>
```
[the remark this answers is not in the document]
Comment (yevhenii-nadtochii) [reply pom.xml:56]: @sonatype-lift ignore

**Review by armiol — APPROVED** [review]
@yevhenii-nadtochii LGTM

As discussed, we are going to ignore the vulnerabilities detected by Sonatype Lift, since we already use the latest available versions of the libraries. Hopefully, in their next updates, they will migrate to the vulnerability-free code.

@alexander-yevsyukov FYI.

**Pull request merged**